package.json

```diff
@@ -104,18 +108,21 @@
     "jest": "^29.6.2",
     "jest-environment-jsdom": "^29.6.2",
     "lint-staged": "^13.2.3",
+    "next": "^13.4.19",
     "postcss": "^8.4.27",
     "prettier": "^3.0.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-icons": "^4.10.1",
-    "storybook": "^7.3.1",
+    "storybook": "^7.3.2",
     "storybook-dark-mode": "^3.0.1",
     "ts-node": "^10.9.1",
+    "tsconfig-paths-webpack-plugin": "^4.1.0",
     "tsup": "^7.2.0",
     "typescript": "^5.1.6",
     "vite": "^4.4.9",
     "vite-tsconfig-paths": "^4.2.0",
```

Comment:
We can probably remove these dependencies too now, yeah?

Reply:
Ahhh yes! We should be able to... as long as we can get the builds to not fail lmao.